src/core/Fbo.tsx

```diff
@@ -1,27 +1,15 @@
 import * as React from 'react'
 import * as THREE from 'three'
 import { useThree } from '@react-three/fiber'
+import { forwardRef } from 'react'
+
+type WebGLRenderTargetCtorParams = ConstructorParameters<typeof THREE.WebGLRenderTarget>
+type WebGLRenderTargetOptions = WebGLRenderTargetCtorParams[2]
 
 type FBOSettings = {
-  /** Defines the count of MSAA samples. Can only be used with WebGL 2. Default: 0 */
-  samples?: number
-  /** If set, the scene depth will be rendered into buffer.depthTexture. Default: false */
+  /** @deprecated use `depthBuffer` instead. If set, the scene depth will be rendered into buffer.depthTexture. Default: false */
```

Comment:
`depth` has the same purpose as `depthBuffer` => deprecate (still backward-compatible)